Description: Relax dep on trybuild
 Upstream pinned it to =1.0.0, which is "for MSRV"; arguably, trybuild has a
 MSRV of 1.34 when it first added rust-version in Cargo.toml, and proptest
 itself has a MSRV of 1.64.
Bug: https://github.com/proptest-rs/proptest/issues/475
Last-Update: 2024-06-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -138,3 +138,3 @@
 [dev-dependencies.trybuild]
-version = "=1.0.0"
+version = "1.0.0"
 
